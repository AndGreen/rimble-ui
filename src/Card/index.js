import React from 'react';
import styled from 'styled-components'

import Box from '../Box'

const Card = styled(Box)`


  ${'' /* background: #FFFFFF; */}
  border: 1px solid #D6D6D6;
  box-shadow: 0px 8px 16px rgba(0, 0, 0, 0.1);
`

Card.defaultProps = {
  bg: 'white',
  mx: 0,
  my: 2,
  width: '100%',
  minWidth: 1,
  height: 'auto',
  minHeight: 1,
  p: 4,
}

Card.displayName = 'Card'

export default Card;
