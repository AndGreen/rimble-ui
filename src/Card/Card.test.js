import Card from './'

describe('Card component sanity', () => {
  it('has name', () => {
    expect(Card.displayName).toBe('Card')
  })

  it('matches default snapshot', () => {
    const component = render(<Card />)
    expect(component).toMatchSnapshot()
  })
})
