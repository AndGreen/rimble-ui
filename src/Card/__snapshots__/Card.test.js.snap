// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component sanity matches default snapshot 1`] = `
.c0 {
  margin-left: 0px;
  margin-right: 0px;
  margin-top: 8px;
  margin-bottom: 8px;
  padding: 32px;
  background-color: white;
  width: 100%;
  min-width: 1px;
  height: auto;
  min-height: 1px;
  position: relative;
  border: 1px solid;
  border-color: #D6D6D6;
  text-align: left;
  text-align: left;
}

.c0 {
  box-sizing: border-box;
}

.c0 {
  box-shadow: 0px 8px 16px rgba(0,0,0,0.1);
  -webkit-transition: box-shadow 300ms ease;
  transition: box-shadow 300ms ease;
}

.c0:hover {
  box-shadow: 0px 8px 16px rgba(0,0,0,0.3);
}

<div
  class="c0"
  height="auto"
  width="100%"
>
  
</div>
`;
